newIDE/app/src/SceneEditor/index.js

```diff
@@ -160,6 +164,7 @@ type State = {|
 
   // State for tags of objects:
   selectedObjectTags: SelectedTags,
+  layoutName: () => string,
```

Comment:
Here you're saying "layoutName is a function that when called is returning a string". 
This seems suspicious, and I don't see any "layoutName" being used in this component `this.state` => you probably don't need this